3.process-cfret-features/scripts/extract_image_features.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Process image features from CellProfiler readout - Only Factin_Adjusted
+# 
+# After discussion, we want to look into the Factin_Adjusted plate as it contains the better protocols for the actin channel so we are more interested in this data.
+# In this notebook, we create two different CSV outputs, one with the image features (e.g., Granularity, Texture, etc.) and one for the image quality metrics (e.g. blur).
+
+# ## Import Libraries
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
+
+from pycytominer import annotate
+from pycytominer.cyto_utils import output
+
+import sys
+sys.path.append("../../utils")
+import extraction_utils as extract_utils
+
+
+# ## Set up paths to CellProfiler directory and outputs
+
+# In[2]:
+
+
+# Set file and directory constants
+cp_dir = "../2.cellprofiler_processing"
```

Comment:
I'm not sure you can use `pathlib` here (looks like it might not work with what you are trying to do), but if you can consider using it to make it clear that this is a path!